modules/openy_gc_auth/modules/openy_gc_auth_reclique/src/EventSubscriber/GCAuthReCliqueUserLoginSubscriber.php

```diff
@@ -66,14 +66,21 @@ public function onUserLogin(GCUserLoginEvent $event) {
             }
           }
           $user_membership = $event->extraData['member']['PackageName'];
```

Comment:
`$user_membership` can be a null or empty string? In this case, we will have `empty($active_roles)` and the same problem that was discussed before. I checked the code and we have the same situation in `openy_gc_auth_yusa`

cc @retif @anpolimus

Reply:
```php
          if ($user_membership && empty($active_roles)) {
            $active_roles = ['virtual_y'];
          }
```

Let's add this condition

Reply:
That works - user is getting authenticated with Virtual Y role assigned.